### ISSUE TITLE
Accessibility - WCAG 2.4.1.Bypass Blocks, skiplinks missing

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#143]
The are no skiplinks. See CMS-Imperia, e.g.  www.efd.admin.ch/  it's easy to integrate
`<li><a class="sprungunsichtbar" href="#webNavigationDiv" ac-cesskey="1">Zur Navigation</a></li><li><a class="sprungunsichtbar" href="#contentStart" accesskey="2">Zum Inhalt</a></li>`   Etc.etc

Original version not ok.
